Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -575,16 +575,15 @@ Renderer::Renderer()
   }
   else
   {
-    if (strstr(g_ogl_config.glsl_version, "1.00") || strstr(g_ogl_config.glsl_version, "1.10") ||
-        strstr(g_ogl_config.glsl_version, "1.20"))
+    if (GLExtensions::Version() < 300)
     {
       PanicAlert("GPU: OGL ERROR: Need at least GLSL 1.30\n"
                  "GPU: Does your video card support OpenGL 3.0?\n"
                  "GPU: Your driver supports GLSL %s",
                  g_ogl_config.glsl_version);
```

Comment:
Why keep this variable around at all? Is it used for anything other than this one error message?